webapp/channels/src/components/latex_block/latex_block.tsx

```diff
@@ -2,74 +2,64 @@
 // See LICENSE.txt for license information.
 
 import type {KatexOptions} from 'katex';
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import {FormattedMessage} from 'react-intl';
 
 type Katex = typeof import('katex');
 
 type Props = {
     content: string;
-    enableLatex: boolean;
+    enableLatex?: boolean;
 };
 
-type State = {
-    katex?: Katex;
-}
+const LatexBlock = ({content, enableLatex}: Props) => {
```

Comment:
Let's separate each prop in a different line. In the future, it helps to verify when we are adding/removing props.
```suggestion
const LatexBlock = ({
    content,
    enableLatex,
}: Props) => {
```